docs/mkdocs.yml

```diff
@@ -0,0 +1,45 @@
+# Project information
+site_name: Gluu Flex 1.0 Docs
```

Comment:
Continuing on the versioning to fit with services like Casa and to probably make more sense moving from 4.x perhaps this should be 5.0 ?

Reply:
Corrected to 5.0